README.md

```diff
@@ -154,6 +154,7 @@ However, we emphasize that these ports are by developers outside the
 libphonenumber project. We do not evaluate their quality or influence their
 maintenance processes.
 
+*   PostgreSQL in-database types (pg-libphonenumber): https://github.com/blm768/pg-libphonenumber
```

Comment:
Please remove the (pg-libphonenumber) part to be consistent with other lines (we don't repeat the name of the repo)